eng/pipelines/aggregate-reports.yml

```diff
@@ -128,3 +94,38 @@ jobs:
       condition: succeededOrFailed()
       displayName: 'Publish Report Artifacts'
       artifact: reports
+  - job: Compliance
```

Comment:
Maybe we should call this job "Analyze (no build)" and the other "Analyze (with build)" or something along those lines. Note from https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#job you can also add a displayName property to be able to use spaces and special chars in the name.